ui/src/infra-model/view/infra-model-view-container.tsx

```diff
@@ -19,53 +21,46 @@ type InfraModelViewContainerProps = {
 export const InfraModelViewContainer: React.FC<InfraModelViewContainerProps> = ({
     viewType,
 }: InfraModelViewContainerProps) => {
+    const navigate = useAppNavigate();
+
     const infraModelState = useInfraModelAppSelector((state) => state);
     const changeTimes = useCommonDataAppSelector((state) => state.changeTimes);
 
     const delegates = createDelegates(infraModelActionCreators);
 
-    const delegatesProps = {
-        onInfraModelExtraParametersChange: delegates.onInfraModelExtraParametersChange,
-        onInfraModelOverrideParametersChange: delegates.onInfraModelOverrideParametersChange,
-        onPlanUpdate: delegates.onPlanUpdate,
-        onPlanFetchReady: delegates.onPlanFetchReady,
+    const [isLoading, setLoading] = React.useState(false);
+
+    const generalProps: InfraModelBaseProps = {
```

Comment:
Tätä ei näytettäisi tarvivan itsessään missään myöhemmin, joten tämän ja `loaderProps`:in luonnin voisi yhdistää yhdeksi objektin luonniksi